content/faculty/honours_board.md

```diff
@@ -6,13 +6,19 @@ submenu_index: 4
 ---
 
 <ul class="nav nav-tabs" id="honours-board-tab">
-  <li class="active"><a href="#staff" data-toggle="tab">Сотрудники</a></li>
-  <li><a href="#students" data-toggle="tab">Студенты</a></li>
-  <li><a href="#sport-heroes" data-toggle="tab">«Герои спорта»</a></li>
+  <li class="active default-tab"><a href="#staff" data-toggle="tab">Сотрудники</a></li>
+    <li><a href="#students" data-toggle="tab">Студенты</a></li>
+    <li><a href="#sport-heroes" data-toggle="tab">«Герои спорта»</a></li>
+    <li class="pull-right">
+      <select class="honour-board-select">
+        <option data-display="Select" disabled>Год</option>
```

Comment:
It seems like the `option` tag is being used in a semantically incorrect way here. Use `optgroup` instead like this:
```html
<optgroup label="Год">
  <option value="2016">2016</option>
  ...
</optgroup>
```